cardano-cli/src/Cardano/CLI/Compatible/Transaction.hs

```diff
@@ -279,69 +277,29 @@ runCompatibleTransactionCmd
         )
         sbe
 
-    let certsRefInputs =
```

Comment:
I assume we don't need reference inputs because QA does not use them?

Reply:
We do - but removed code was only used to build a body only for creating signatures. But it's problematic: that's a slightly different body than the one we're using in building compatible transaction. So the transaction building logic is duplicated in two places, and we had the bug with incorrect body hash used for the signatures.